pkgs/top-level/all-packages.nix

```diff
@@ -371,7 +371,7 @@ with pkgs;
 
   buildEnv = callPackage ../build-support/buildenv { }; # not actually a package
 
-  buildFHSEnv = buildFHSEnvBubblewrap;
+  buildFHSEnv = if stdenv.hostPlatform.isx86 then buildFHSEnvBubblewrap else buildFHSEnvChroot;
   buildFHSEnvChroot = callPackage ../build-support/build-fhsenv-chroot { }; # Deprecated; use buildFHSEnv/buildFHSEnvBubblewrap
```

Comment:
`buildFHSEnvChroot` is still... deprecated. Also, ofborg does not pass.

Would you please check if #227510 works? I created that PR in the context of `nextflow`, but `boinc` might also work.

Reply:
That PR will land in nix{pkgs,os}-unstable in a few days at most: https://status.nixos.org